contracts/strategies/ltip-simple/LTIPSimpleStrategy.sol

```diff
@@ -0,0 +1,708 @@
+            if (votes[recipientId] < votingThreshold) revert INSUFFICIENT_VOTES();
+
+            if (_vestingPlans[recipientId].vestingContract != address(0)) revert ALREADY_ALLOCATED();
+
+            // Get the pool, subtract the amount and transfer to the recipient
+            poolAmount -= recipient.allocationAmount;
+
+            _vestAmount(recipientId, recipient.recipientAddress, pool.token, recipient.allocationAmount);
+
+            // Emit events for the milestone and the distribution
+            emit Distributed(recipientId, recipient.recipientAddress, recipient.allocationAmount, _sender);
+
+            unchecked {
+                ++i;
+            }
+        }
+    }
+
+    function _isPoolTimestampValid(
+        uint64 _allocationStartTime,
+        uint64 _allocationEndTime,
+        uint64 _distributionStartTime,
+        uint64 _distributionEndTime,
+        uint64 _registrationStartTime,
+        uint64 _registrationEndTime,
+        uint64 _reviewStartTime,
+        uint64 _reviewEndTime
+    ) internal view {
+        if (
+            block
+                // Register timestamps must be in the future
+                .timestamp > _registrationStartTime
+            // Start times must be before end times
+            || _allocationStartTime > _allocationEndTime || _distributionStartTime > _distributionEndTime
+                || _registrationStartTime > _registrationEndTime || _reviewStartTime > _reviewEndTime
+            // Some end times must be after other end times
+            || _registrationEndTime > _allocationEndTime || _allocationEndTime > _distributionStartTime
+                || _reviewStartTime > allocationEndTime
+        ) {
+            revert INVALID();
+        }
+    }
+
+    /// @notice Check if sender is a profile owner or member.
+    /// @param _anchor Anchor of the profile
+    /// @param _sender The sender of the transaction
+    /// @return 'true' if the sender is the owner or member of the profile, otherwise 'false'
+    function _isProfileMember(address _anchor, address _sender) internal view returns (bool) {
+        IRegistry.Profile memory profile = _registry.getProfileByAnchor(_anchor);
+        return _registry.isOwnerOrMemberOfProfile(profile.id, _sender);
+    }
+
+    /// @notice Checks if address is eligible allocator.
+    /// @dev This is used to check if the allocator is a pool manager and able to allocate funds from the pool
+    /// @param _allocator Address of the allocator
+    /// @return 'true' if the allocator is a pool manager, otherwise false
+    function _isValidAllocator(address _allocator) internal view virtual override returns (bool) {
+        return allo.isPoolManager(poolId, _allocator);
+    }
+
+    /// @notice Get the recipient.
+    /// @param _recipientId ID of the recipient
+    /// @return recipient Returns the recipient information
+    function _getRecipient(address _recipientId) internal view returns (Recipient memory recipient) {
```

Comment:
This is not really being utilized that much other than in `_getRecipientStatus`. We can either utilize it more in other parts of the code or delete it.

Reply:
gone :)